README.md

```diff
@@ -25,32 +15,84 @@ Install as usual, see
 CONFIGURATION
 -------------
 
-The module has a simple configuration screen, where you provide Server Key and
-Firebase's endpoint.
+The module has a simple configuration form, where you provide Server Key and
+Sender ID.
+
+HOW TO USE
+-----------
 
-* In order to send the push notification:
+In order to send the push notification message with data:
 
 ```php
 // Token is generated by app. You'll have to send the token to Drupal.
 $fakeToken = 'e3vUiwcvkpY:APA91bEDZzKTIkaL0e-UTwiV6EGi1m5J5PrDMxejm6-d85vdwAgd';
-\Drupal::service('firebase.notification')->send($fakeToken, [
+$messageService = \Drupal::service('firebase.message');
+$messageService->addRecipients($fakeToken);
+$messageService->addNotification([
   'title' => 'Title goes here',
   'body' => 'Body goes here',
-  'data' => [
```

Comment:
so this require to rework all code which using old method, and there's no BC kept